README.md

```diff
@@ -763,27 +763,27 @@ Run `mbed update <tag_name|revision> --clean`
 
 __When you have unpublished local libraries__
 
-There are three additional options that define how unpublished local libraries are handled:
+There are three additional options that show how unpublished local libraries are handled:
```

Comment:
Might just be me but when I see `show` coupled with an option description, I assume it's going to print something to screen. When really these options modify the behavior of the tool. Perhaps we can say `modify` instead of `define` or `show`?